README.md

```diff
@@ -116,9 +116,13 @@ From root repo directory:
 
 After running through the config setup UI flow once, you will need to delete the `fedimintd` data to run through it again. To do this, delete the `fm_1`, `fm_2`, `fm_3`, and `fm_4` folder from the repo. These are data directories mounted to Docker containers running fedmintd and are listed in `.gitignore` so are safe to remove.
 
-### Running with mprocs
+### Running with Nix and mprocs
 
 1. Install [mprocs](https://github.com/pvolok/mprocs)
+1. Install [Nix](https://nixos.org/download)
+1. Add `experimental-features = nix-command flakes` to your `/etc/nix/nix.conf` file
+1. In a terminal `cd` to the fedimint-ui repo root directory
+1. Run `nix develop`
 1. Run `mprocs -c mprocs.yml`
```

Comment:
This is actually a docker-centric process, the `nix` way of doing development is using the `yarn nix-guardian` and `yarn nix-gateway` commands. Those leverage `devimint` on your machine, they aren't using containers. The only value with of using nix with this set of instructions is that you wouldn't need to install `mprocs`.

The hope for #217 was for documenting usage of those `yarn nix-*` commands, which we have some light notes for in `### Running with local Fedimint`.

It might be wise to split the `## Development` section into two completely separate ones, one with nix and one with docker.

Reply:
Got it. I'll keep at it. Thanks for the feedback. I'm still getting up to speed with Nix, but so too are many that will be reading this document, so hopefully that's a good thing.

Reply:
yeah the steps are:

1. install nix
2. nix develop (just put up a pr that will also run yarn install when you do this so the build env is complete at this point)
3. yarn nix-gateway or yarn nix-guardian . Big difference is nix-gateway preconfigures the federation so you don't have to go through federation setup, nix-guardian starts the separate guardian nodes that you connect into a federation via the setup process. 

Will just added guardian UIs to `yarn nix-gateway` so you can see the guardian dashboards without doing setup, so might be worth renaming them to be more clear.

Reply:
I think I got everything as you wanted. Appreciate the opportunity to help out. If you need me to dig into anything else I am here to serve.

Reply:
Kody, you beat me to the yarn install. You da man